include/oneapi/dnnl/dnnl_graph.hpp

```diff
@@ -846,6 +846,8 @@ class op : public op_handle {
         TanhBackward = dnnl_graph_op_tanh_backward,
         TypeCast = dnnl_graph_op_type_cast,
         Wildcard = dnnl_graph_op_wildcard,
+        GenIndex = dnnl_graph_op_gen_index,
+        GreaterEqual = dnnl_graph_op_greater_equal,
```

Comment:
I think we can add the pattern document in another PR, but the op document better to be included here.

Reply:
OK, will do